docs/iana-application.md

```diff
@@ -0,0 +1,55 @@
+### Transport protocols
+- [x] TCP
+- [x] UDP
+- [ ] SCTP
+- [ ] DCCP
+
+### Service Name
+StatsD
+
+### Desired Port Number
+8125
+
+### Description
+Network daemon for collecting metrics and publishing to various metric aggregation systems or services.
+
+### Reference
+StatsD is used to collect statistics such as counters and timers to measure system and application performance. StatsD receives metrics over UDP or TCP in a basic line protocl of the following format:
+
+`<metricname>:<value>|<type>`
+
+The metric name (or "bucket") is an identifier for the metric being sent. Values represent the metric value being pushed to StatsD and should be of the specified "type". Statsd supports numerous mertric types such as counters, gagues, and timers. Each has it's own single character "short code" key which is used in the type section.
+
+Metrics are collected and flushed at a desired interval to the plugable backend for storage and analysis.
+
+Other projects and organisations have also created extensions to the StatsD protocol, one popular example being DogStatsD by DataDog. See https://docs.datadoghq.com/developers/dogstatsd/ for more details on the extension.
+
+### If UDP is requested, please explain how traffic is limited, and whether the protocol reacts to congestion.
+Statsd does not react to congestion itself, but does provide the means for creating StatsD clusters to aid performance.
```

Comment:
Consistently upcase the `D` in `StatsD`?

Reply:
My fine and replace foo failed me